### PR TITLE
Added a sentence to `binstubs`

### PR DESCRIPTION
**Pull request by rubymorillo**
Per feedback from @arbonap, we've added a sentence that briefly explains what binstubs is to the binstubs section.

Thanks so much for the contribution!
To make reviewing this PR a bit easier, please fill out answers to the following questions.

### What was the end-user problem that led to this PR?

The problem was...no understanding of what binstubs is. Added a quick two sentences!

### What was your diagnosis of the problem?

My diagnosis was...we needed some language there. @arbonap and @indirect pointed this out and offered the explanations.

### What is your fix for the problem, implemented in this PR?

My fix...was to turn their explanations into simple language!

### Why did you choose this fix out of the possible options?

I chose this fix because...it'll help users understand binstubs better

### CODE REVIEW

File: man/bundle-install.ronn [hunk 1]
```diff
@@ -47,7 +47,12 @@ To apply any of `--deployment`, `--path`, `--binstubs`, or `--without` every
 time `bundle install` is run, use `bundle config` (see bundle-config(1)).
 
 * `--binstubs[=<directory>]`:
-  Creates a directory (defaults to `~/bin`) and place any executables from the
+  Binstubs are scripts that wrap around executables. Bundler creates a small Ruby
+  file (a binstub) that loads Bundler, runs the command, and puts it in `bin/`.
```
Comment (colby-swandale) [inline man/bundle-install.ronn]: Just need to remove the spaces at the end of the line.

File: man/bundle-install.ronn [hunk 2]
```diff
@@ -47,7 +47,12 @@ To apply any of `--deployment`, `--path`, `--binstubs`, or `--without` every
 time `bundle install` is run, use `bundle config` (see bundle-config(1)).
 
 * `--binstubs[=<directory>]`:
-  Creates a directory (defaults to `~/bin`) and place any executables from the
+  Binstubs are scripts that wrap around executables. Bundler creates a small Ruby
+  file (a binstub) that loads Bundler, runs the command, and puts it in `bin/`. 
+  This lets you link the binstub inside of an application to the exact gem
+  version the application needs.
+
```
Comment (colby-swandale) [inline man/bundle-install.ronn]: and here as well

**Comment by rubymorillo** [comment]
@colby-swandale removed the whitespace. I'm new at this, so pls let me know if there are any more problems :) . Thank you!

File: man/bundle-install.ronn [hunk 3]
```diff
@@ -47,7 +47,12 @@ To apply any of `--deployment`, `--path`, `--binstubs`, or `--without` every
 time `bundle install` is run, use `bundle config` (see bundle-config(1)).
 
 * `--binstubs[=<directory>]`:
-  Creates a directory (defaults to `~/bin`) and place any executables from the
+  Binstubs are scripts that wrap around executables. Bundler creates a small Ruby
+  file (a binstub) that loads Bundler, runs the command, and puts it in `bin/`.
+  This lets you link the binstub inside of an application to the exact gem
+  version the application needs.
+
```
Comment (colby-swandale) [inline man/bundle-install.ronn]: There is a whitespace here as well that needs to be removed.

**Comment by colby-swandale** [comment]
If you would like to verify that your change before pushing to CI, run `rake spec:deps && bin/rspec spec/quality_spec.rb`

**Comment by segiddins** [comment]
I think this just needs to pass the quality specs and then we can merge

**Comment by colby-swandale** [comment]
ping @rubymorillo are you still working on this?

File: man/bundle-install.ronn (hunk 3 above)
Comment (rubymorillo) [reply man/bundle-install.ronn]: @colby-swandale I'm not seeing the extra whitespace—mind pointing it out?

Comment (colby-swandale) [reply man/bundle-install.ronn]: There is a space character on line 54 that shouldn't be there (which is why the build is failing in CI)

**Comment by colby-swandale** [comment]
This is failing in CI because there was 2 space characters on line 54, you removed one but both need to be removed. Thanks

**Comment by colby-swandale** [comment]
@bundlerbot r+

**Pull request merged**